pkgs/rust-internal-deps/default.nix

```diff
@@ -41,6 +41,8 @@ let
       # a) bundled at buildtime if possible (binaries/packages)
       # b) use this version of nixpkgs i.e. (nix library utils such as `lib` and `runCommand`)
       COMMON_NIXPKGS_URL = "github:flox/nixpkgs/${inputs.nixpkgs.rev}?narHash=${inputs.nixpkgs.narHash}";
+      # Some tests need a URL in https format
+      TESTING_COMMON_NIXPKGS_URL = "https://github.com/flox/nixpkgs?rev=${inputs.nixpkgs.rev}";
```

Comment:
It's arguably only the nef build things that so far have to hardcode a url.
Besides that url is not even a valid flakeref unlike `COMMON_NIXPKGS_URL`, which is also used by tests (nef unit tests).

I'd suggest renaming this to `TESTING_BASE_CATALOG_URL`

Reply:
Renamed